lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -2035,8 +2118,8 @@ class CommandObjectTargetModulesDumpSymtab
                 result.GetOutputStream().EOL();
                 result.GetOutputStream().EOL();
               }
-              if (INTERRUPT_REQUESTED(GetDebugger(), 
```

Comment:
It would be nice to remove these space only changes. Ditto for many space only changes below.

Reply:
Done. Turns out my editor (correctly) had its "remove trailing whitespace on save" setting set.